daemon/src/connection.rs

```diff
@@ -460,14 +418,8 @@ impl Actor {
                     tracing::warn!(%order_id, "No active setup actor");
                 }
             }
-            wire::MakerToTaker::Settlement { order_id, msg } => {
-                if let Err(NotConnected(_)) = self
-                    .collab_settlement_actors
-                    .send_async(&order_id, msg)
-                    .await
-                {
-                    tracing::warn!(%order_id, "No active collaborative settlement")
-                }
+            wire::MakerToTaker::Settlement { .. } => {
+                tracing::error!("legacy handler - use libp2p instead");
```

Comment:
❓ When should we remove this? Why not now?

Reply:
I'd refrain to do so until we fully drop our legacy networking; then the whole enum will go away.